docs/high_availability_installation_guide.md

```diff
@@ -1,30 +1,21 @@
 # Harbor High Availability Guide 
 
```

Comment:
What's this document for? If it's only an simple HA set up, I think it's not necessary to describe what are the stateful and stateless services in Harbor? This is not enough to let reader knows the technical details but introduce more questions.

Reply:
It's help user to better under Harbor and can be used as a reference for user to setup HA. This document also give an example on how to setup HA.